tox.ini

```diff
@@ -337,8 +337,8 @@ commands=
   bash {toxinidir}/tests/scripts/vagrant_up.sh --no-provision {posargs:--provider=virtualbox}
   bash {toxinidir}/tests/scripts/generate_ssh_config.sh {changedir}
 
-  # configure lvm
-  !lvm_batch-!lvm_auto_discovery: ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/tests/functional/lvm_setup.yml
+  # configure lvm, we exclude osd2 given this node uses lvm batch scenario (see corresponding inventory host file)
+  !lvm_batch-!lvm_auto_discovery: ansible-playbook -vv -i {changedir}/{env:INVENTORY} {toxinidir}/tests/functional/lvm_setup.yml --limit 'osds:!osd2'
```

Comment:
we need this in the `[purge]` section too

https://github.com/ceph/ceph-ansible/blob/master/tox.ini#L60